### PR TITLE
XIVY-13820 doc: form-editor news for 11.3

### PR DESCRIPTION
**Pull request by ivy-rew**
feel free to browse on my machine: http://192.168.3.99/news/11.3
form-editor

### CODE REVIEW

File: src/app/pages/news/11.3/07-form-editor.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+We are enabling you to build user interfaces like never before.
+Looking for feedback, we already integrated an early Alpha version of our new Form Editor.
+
+- **Low-Code**: Committed to let everyone partake in building UI, the editor let's you define contents without writing any code. Drag and Drop skills is all you need to start your creative journey.
+- **Agnostic**: Building the UI in technology neutral manor, we automatically generated the views for JSF as before. However, you're no longer the maintainer of it and can ignore any efforts to make it work and keep it stable.
+- **Alpha**: We plan to support much more widgets, and integrations of complex components until the LTS 12 release.
+
+<div class="short-links">
+	<a href="${docBaseUrl}/designer-guide/user-interface/user-dialogs/html-dialog-editor.html"
```
Comment (ivy-rew) [inline src/app/pages/news/11.3/07-form-editor.md:11]: this doc does not yet exist: I plan to initiate it, just as we did for the existing html-dialog-editor https://dev.axonivy.com/doc/11.3/designer-guide/user-interface/user-dialogs/html-dialog-editor.html

**Review by ivy-lli — APPROVED** [review]
Looks good to me 👍

File: src/app/pages/news/11.3/07-form-editor.md (hunk 1 above)
Comment (ivy-rew) [reply src/app/pages/news/11.3/07-form-editor.md:10]: initial docs are on the way: https://github.com/axonivy/core/pull/5748

File: src/app/pages/news/11.3/07-form-editor.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Form Editor (Alpha) 🧪 {#formEditor}
+
+We are enabling you to build user interfaces like never before.
+
+- **Low-Code**: Committed to let everyone partake in building UIs, the new Form Editor let's you define contents without writing any code. Drag and Drop skills is all you need to start your creative journey.
```
Comment (weissreto) [inline src/app/pages/news/11.3/07-form-editor.md:5]: let -> letting
let's -> lets
skills is -> skills are | skill is

File: src/app/pages/news/11.3/07-form-editor.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- **Agnostic**: Building the UI in technology neutral manor, we automatically generate the views for JSF as before. However, you're no longer the maintainer of them and can refrain from efforts to make them functional and keeping them stable.
```
Comment (weissreto) [inline src/app/pages/news/11.3/07-form-editor.md:6]: technology neutral -> technology-neutral
manor -> mannor
keeping -> keep

**Review by weissreto — APPROVED** [review]
Grammery to the rescue

**Pull request merged**